testing/apache2-mod-security/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Ondrej Esler <ondrej.esler@intraworlds.com>
+# Maintainer: Ondrej Esler <ondrej.esler@intraworlds.com>
+pkgname=apache2-mod-security
+_pkgreal=modsecurity
+_realname=mod_security2
+pkgver=2.9.3
+pkgrel=0
+pkgdesc="Security Module for Apache2"
+url="https://www.modsecurity.org/"
+arch="all"
+license="Apache-2.0"
+depends="apache2 pcre libxml2"
```

Comment:
I bet `pcre libxml2` will be added by build system... see in travis
```
...
>>> apache2-mod-security*: Tracing dependencies...
	apache2
	pcre
	libxml2
	so:libapr-1.so.0
	so:libaprutil-1.so.0
	so:libc.musl-x86_64.so.1
	so:libpcre.so.1
	so:libxml2.so.2
```

Also not clear why travis reports `>>> WARNING: apache2-mod-security*: Redundant /usr/lib in rpath found`